notebooks/getting_started/bq_dataframes_template.ipynb

```diff
@@ -114,7 +114,7 @@
     },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    df.sort_values("body_mass_g").plot(title="Numeric features", logy=True)
</pre><p> </p><p>Might need a <code>reset_index()</code> call in strict ordering mode for the sort to make any difference there. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/1665/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
Added `reset_index` in latest commit.